### ISSUE TITLE
Placeholder shows under transparent background images

### ISSUE THREAD

**Issue by MrPetovan** [friendica/friendica#13466]
Follow-up to #13455

Example: https://friendica.mrpetovan.com/display/cb811c25-8d4bf46842f7b8b5-5e948df7
Original post: https://akko.wtf/notice/Aa4dk4sGBhWtNma7sG

This is a tricky one, as the placeholder is created using only HTML/CSS, which means we can’t react to an image fully loading by removing the placeholder dynamically without adding some Javascript which I’d prefer to avoid.

**Comment by fabrixxm** [comment]
imho, the only option is to remove the css background from `<figure>` element.

Maybe we could do some magic trick with "[srcset](https://developer.mozilla.org/en-US/docs/Web/API/HTMLImageElement/srcset)" property on img tag?

**Comment by MrPetovan** [comment]
> imho, the only option is to remove the css background from `<figure>` element.

Indeed, but we need something to show as placeholder because we allocate the image space. Without anything, it looks weird.

> Maybe we could do some magic trick with "[srcset](https://developer.mozilla.org/en-US/docs/Web/API/HTMLImageElement/srcset)" property on img tag?

I leave magic to wizards.

**Comment by fabrixxm** [comment]
I've done some tests. Looks like the only way to have image placeholders that work with transparent images is to use js

**Comment by MrPetovan** [comment]
Thank you for looking into it!
